packages/legend-query-builder/src/application/QueryBuilderCache.ts

```diff
@@ -14,22 +14,8 @@
  * limitations under the License.
  */
 
-import {
-  LegendApplicationPlugin,
-  type LegendApplicationPluginManager,
-} from '@finos/legend-application';
-import packageJson from '../../package.json';
-
-export class DSL_DataSpace_LegendApplicationPlugin extends LegendApplicationPlugin {
-  static NAME = packageJson.extensions.applicationPlugin;
-
-  constructor() {
-    super(DSL_DataSpace_LegendApplicationPlugin.NAME, packageJson.version);
-  }
-
-  install(
-    pluginManager: LegendApplicationPluginManager<LegendApplicationPlugin>,
-  ): void {
-    pluginManager.registerApplicationPlugin(this);
-  }
+export enum QUERY_BUILDER_CACHE_KEY {
```

Comment:
--> `QUERY_BUIDER_USER_DATA_KEY`

and rename this file to `QueryBuilderUserData`